### PR TITLE
feat(default_icon): expose `default_icon` through `get_default_icon`

### PR DESCRIPTION
**Pull request by utilyre**

### CODE REVIEW

File: lua/nvim-web-devicons.lua [hunk 1]
```diff
@@ -1704,14 +1704,18 @@ local function setup(opts)
   })
 end
 
+local function get_default_icon(opts)
```
Comment (alex-courtis) [inline lua/nvim-web-devicons.lua:1707]: Thank you for raising this one.

Regular use cases following the refactor work just fine.

I did have to set default when calling get_default_icon. Was that your intention? I would expect that calling it with no args would return the default.

```lua
local wdi = require("nvim-web-devicons")

local def = wdi.get_default_icon()
print("opts = {} : " .. vim.inspect(def))

def = wdi.get_default_icon({ default = true })
print("opts = { default = true } : " .. vim.inspect(def))
```

```
opts = {} : nil
opts = { default = true } : {
  color = "#6d8086",
  cterm_color = "66",
  icon = "",
  name = "Default"
}
```

**Review by alex-courtis — CHANGES_REQUESTED** [review]

File: lua/nvim-web-devicons.lua (hunk 1 above)
Comment (utilyre) [reply lua/nvim-web-devicons.lua:1707]: I thought it'd be a good opportunity to remove duplicate code ([65d7182](https://github.com/nvim-tree/nvim-web-devicons/pull/186/commits/65d71827b418ce2208d622bc529f829d9f1e6bdb)). But now that you mention this odd behavior, I guess just returning the default_icon is good enough.

Comment (utilyre) [reply lua/nvim-web-devicons.lua]: Just removed the opts param.

**Review by alex-courtis — APPROVED** [review]
Many thanks for your contribution.

**Pull request merged**